xarray/core/accessor_str.py

```diff
@@ -883,6 +1105,45 @@ def count(
         Returns
         -------
         counts : array of int
+
+        Examples
+        --------
+        >>> da = xr.DataArray(["jjklmn", "opjjqrs", "t-euuJJvwx"], dims="x")
+        >>> da
+        <xarray.DataArray (x: 3)>
+        array(['jjklmn', 'opjjqrs', 't-euuJJvwx'], dtype='<U10')
+        Dimensions without coordinates: x
+
+        Using a string:
+        >>> da.str.count("jj")
+        <xarray.DataArray (x: 3)>
+        array([1, 1, 0])
+        Dimensions without coordinates: x
+
+        Enable case-insensitive matching by setting case to false:
+        >>> counts = da.str.count("jj", case=False)
+        >>> counts
+        <xarray.DataArray (x: 3)>
+        array([1, 1, 1])
+        Dimensions without coordinates: x
+
+        Using regex:
+        >>> pat = "jj\\w+"
```

Comment:
Maybe we could come up with a more interesting example.
Maybe add a number after one of the "jj" in the DataArray, and then look for "jj\\d"?

But this version is already a good start.

Reply:
Will do, thank you!